tests/nested/manual/hybrid-remodel/task.yaml

```diff
@@ -0,0 +1,61 @@
+summary: Test remodeling on a hybrid system
+
+details: |
+  This test remodels on a hybrid system to install a new kernel snap and new
+  application snaps.
+
+systems: [ubuntu-22.04-64]
```

Comment:
we actually should try to run on 23.10 and 24.04,  for 23.10 there is a classic-23.10/stable track, but Alfonso probably can tell you more concrete things here

Reply:
Doesn't the model determine the nested system's version, rather than the array here?

Reply:
Regardless, @alfonsosanchezbeato, do you have suggestions for systems that this should be tested on?

Reply:
my brain got a bit mixed up because hybrid is classic but I missed that this is a nested test. Anyway Alfonso can help more than me with details here

Reply:
This is fine, as this is indeed nested. See comments below about muinstaller anyway.